test/functional/test_framework/mininode.py

```diff
@@ -77,14 +77,19 @@ def __init__(self):
 
         super().__init__(map=mininode_socket_map)
 
+        self.conn_open = False
+
+    def is_connected(self):
```

Comment:
suggestion: change this to a property (use `@property` decorator)

Reply:
I believe this doesn't work when the implementation details change and we no longer return a member. That is, when we derive the return value with an expression. E.g. `self._transport is not None` or `self._state == State.CONNECTED` or ...

Reply:
It should still work. a `@property` decorated method can do anything any other method can do.

Reply:
Ah thanks for clarifying. Reworked the patch to make it a property.